CMakeLists.txt

```diff
@@ -131,6 +132,22 @@ if(F3D_MODULE_ASSIMP)
   endif()
 endif()
 
+if(F3D_MODULE_ALEMBIC)
+  find_package(Alembic 1.7 REQUIRED)
+  find_package(Imath 3.0 REQUIRED)
```

Comment:
Do you access to `Imath` APIs? If it's a private dependency, you won't need to explicitly require this package.

Reply:
Yes, it is a private dependency. I have removed it.